frontend/packages/ux-editor/src/components/config/editModal/EditOptions/OptionTabs/EditTab/OptionListSelector/OptionListSelector.tsx

```diff
@@ -44,32 +32,33 @@ export function OptionListSelector<T extends SelectionComponentType>({
         />
       );
     case 'error':
-      return (
-        <ErrorMessage>
-          {error instanceof Error ? error.message : t('ux_editor.modal_properties_error_message')}
-        </ErrorMessage>
-      );
+      return <ErrorMessage>{t('ux_editor.modal_properties_error_message')}</ErrorMessage>;
```

Comment:
This is not the correct error message. I think this one is the correct: `ux_editor.modal_properties_fetch_option_list_ids_error_message` 🫣